hdl21/primitives.py

```diff
@@ -686,6 +715,7 @@ def Pnp(arg: Any = Default, **kwargs) -> Primitive:
 class DiodeParams:
     w = Param(dtype=Optional[Scalar], desc="Width in resolution units", default=None)
     l = Param(dtype=Optional[Scalar], desc="Length in resolution units", default=None)
+    a = Param(dtype=Optional[Scalar], desc="Area in resolution units squared", default=None)
```

Comment:
Do we really need/ want this as an independent parameter? 
Is the fact that there are `w` and `l` parameters not implicative that the shape is a rectangle, and `a == w * l`?